src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,69 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.sql.SQLException;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+import mate.academy.util.InitDatabase;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        try {
+            InitDatabase.executeInitScript(ConnectionUtil.getConnection());
+        } catch (SQLException e) {
+            throw new DataProcessingException("Cannot initialize DataBase", e);
```

Comment:
The exception message should be more informative. Include details that could help identify the issue more easily, such as the script name or action being performed when the exception occurred.